app/src/main/java/de/westnordost/streetcomplete/overlays/AbstractOverlayForm.kt

```diff
@@ -347,6 +352,13 @@ abstract class AbstractOverlayForm :
             if (element.isSplittable()) {
                 answers.add(AnswerItem(R.string.split_way) { splitWay(element) })
             }
+
+            if (element is Node // add moveNodeAnswer only if it's a free floating node
+                && mapDataWithEditsSource.getWaysForNode(element.id).isEmpty()
+                && mapDataWithEditsSource.getRelationsForNode(element.id).isEmpty()) { // todo: allow relations except multipolygon?
+                answers.add(AnswerItem(R.string.move_node) { moveNode() })
```

Comment:
I was thinking about that, but decided to use the same as you wanted to avoid using the word _node_, and the only alternative that came in my mind was _point_, which imo doesn't sound right.

And maybe "At slightly wrong _position_" should be changed to "At slightly wrong _location_", as this is also used when clicking _can't say_.

Reply:
Ok